src/components/MerchantPayout/MerchantPayoutScreen.tsx

```diff
@@ -8,7 +8,6 @@ import React, {
 import {
   View,
   StyleSheet,
```

Comment:
In `AllValidVouchersModal` there is an alert that we haven't converted to use the new showXXXAlert~

Reply:
This is done!